politeiawww/paywall.go

```diff
@@ -6,12 +6,22 @@ import (
 	"github.com/decred/politeia/util"
 )
 
+const (
+	mainNetDefaultMinConfirmationsRequired = uint64(2) // Min Confirmations Required for mainnet. Can not be changed
```

Comment:
This should come from config, always. So the proper default should be set there and then copied into whatever variable. Now we have it in 2 places.